code/game/objects/items/storage/belt.dm

```diff
@@ -687,3 +687,27 @@
 /obj/item/storage/belt/sabre/PopulateContents()
 	new /obj/item/melee/sabre(src)
 	update_icon()
+
+/obj/item/storage/belt/military/tactibelt/ComponentInitialize()
+	. = ..()
+	GET_COMPONENT(STR, /datum/component/storage)
+	STR.max_items = 4
+	icon_state : "tactibelt"
```

Comment:
WHY NO JESUS FUCK WHY

Reply:
trolld epicly
:trollface:

Reply:
fr just change the : after the icon_state, item_state and desc with a = and it should work

Reply:
What the FUCK is this shit.